estuary_updater/handlers/distgit.py

```diff
@@ -25,8 +25,7 @@ def can_handle(msg):
         :rtype: bool
         """
         supported_topics = [
-            '/topic/VirtualTopic.eng.distgit.commit',
-            '/topic/VirtualTopic.eng.distgit.push'
```

Comment:
Could you also remove this from `fedmsg.d/example.py`?

Reply:
For whatever reason it's already not in `example.py`, just in case you're looking for this change